trunk/SUAVE/Methods/Weights/Buildups/Electric_Vectored_Thrust/empty.py

```diff
@@ -0,0 +1,156 @@
+
+## @ingroup Methods-Weights-Buildups-Electric_Vectored_Thrust
+def empty(config,
+          speed_of_sound                = 340.294,
+          max_tip_mach                  = 0.65,
+          disk_area_factor              = 1.15,
+          max_thrust_to_weight_ratio    = 1.1,
+          motor_efficiency              = 0.85 * 0.98):
+    """weight = SUAVE.Methods.Weights.Buildups.electricVectored_Thrust.empty(
+            config,
+            speed_of_sound              = 340.294,
+            max_tip_mach                = 0.65,
+            disk_area_factor            = 1.15,
+            max_thrust_to_weight_ratio  = 1.1,
+            motor_efficience            = 0.85 * 0.98)
+        
+        Calculates the empty fuselage mass for an electric tiltrotor including
+        seats, avionics, servomotors, ballistic recovery system, rotor and hub
+        assembly, transmission, and landing gear. Additionally incorporates
+        results of the following correlation scripts:
+            fuselage,py
+            prop.py
+            wing.py
+            wiring.py
+        Originally written as part of an AA 290 project inteded for trade study
+        of the Electric Vectored_Thrust along with the following defined SUAVE vehicle types:
+            Electric Multicopter
+            Electric Stopped Rotor
+
+        Sources:
+        Project Vahana Conceptual Trade Study
+        https://github.com/VahanaOpenSource
+        
+        Inputs:
+            config                          SUAVE Config Data Structure
+            speed_of_sound                  Local Speed of Sound                [m/s]
+            max_tip_mach                    Allowable Tip Mach Number           [Unitless]
+            disk_area_factor                Inverse of Disk Area Efficiency     [Unitless]
+            max_thrust_to_weight_ratio      Allowable Thrust to Weight Ratio    [Unitless]
+            motor_efficiency                Motor Efficiency                    [Unitless]
+            
+        Outputs:
+            output:                         Data Dictionary of Component Masses       [kg]
+    """
+
+
+    
+    #-------------------------------------------------------------------------------
+    # Unpack Inputs
+    #------------------------------------------------------------------------------- 
+    rRotor              = config.propulsors.propulsor.rotor.tip_radius 
+    bladeSol            = config.propulsors.propulsor.rotor.blade_solidity
+    tipMach             = max_tip_mach
+    k                   = disk_area_factor
+    ToverW              = max_thrust_to_weight_ratio
+    etaMotor            = motor_efficiency  
+
+    #-------------------------------------------------------------------------------
+    # Assumed Weights
+    #-------------------------------------------------------------------------------
+    output = Data()
+    output.payload      = config.propulsors.propulsor.payload.mass_properties.mass
+    output.seats        = 30.
```

Comment:
This looks like a lot of the same stuff from at least one of the other buildups. Why hasn't it been combined?

Reply:
See the previous comment